policies/committer-policy.md

```diff
@@ -0,0 +1,112 @@
+
+# Policy for OpenSSL Committers
+
+## Who is a committer?
+
+OpenSSL committers are contributors who have commit access to the OpenSSL
+source code repository.  Committers review and commit their own patches
+as well as those of other contributors.
+
+## How to become a committer?
+
+Commit access is granted by a vote of the [OMC] typically on the
+recommendation of the [OTC] (see the [OpenSSL Bylaws]).
+
+We welcome contributors who become domain experts in some part of
+the library (for example, low-level crypto) as well as generalists
+who contribute to all areas of the codebase.  All committers share
+the responsibility for the overall health of the project: aside from
+contributing quality features, committers are team players who fix bugs,
+address open issues, review community contributions, and improve tests
+and documentation.  Committers are also shepherds of the OpenSSL community
+and its code of conduct.
+
+To become a committer, start by contributing code.  Read our coding style,
+and get to know our build and test system.  Then, use the Github issue
+tracker, and our mailing lists to find impactful ideas to work on.
+
+## How to maintain committer status?
+
+To maintain committer status, you must stay active in the project.  Refer
+to the OpenSSL Bylaws for details.
+
+In the unlikely and unfortunate event that your actions conflict with
+the project objectives or are otherwise disruptive, committer status
+may also be revoked by the OMC.
+
+## Approvals and code reviews
+
+All submissions must be reviewed and approved by at least two committers,
+one of whom must also be an OTC member.  Neither of the reviewers can
+be the author of the submission.
```

Comment:
As a non-OMC member I ask myself: What is the rationale behind this significant tightening of the rules? 🤔

Reply:
+1.

Reply:
Mostly it represents a slackness in the current review process.

Reply:
Well... I'm afraid it will not add much energy to it.

Reply:
It's about improving quality and ensuring enough eyes look at every change.

Reply:
I understand intentions but have doubts about results.

Reply:
I had the same doubts when we introduced the original review process - but I would not go back now. We always have the option of reverting this change if it doesn't work out well.

Reply:
I agree with the original process, but I think this change will not increase the quality but just processing time. 
Could we establish some metrics?

Reply:
Feasibly we could establish metrics for review time (e.g. measure time from PR Open to PR Close). I'm not sure how you would establish metrics for quality.

Reply:
In some specific cases, it would be possible to improve the quality of the review by not just looking at the code but also download and build it locally, and then test it independently of the author. But I doubt that such an option could be turned into in a general requirement.